template.go

```diff
@@ -0,0 +1,108 @@
+package cli
+
+// AppHelpTemplate is the text template for the Default help topic.
+// cli.go uses text/template to render templates. You can
+// render custom help text by setting this variable.
+var AppHelpTemplate = `NAME:
```

Comment:
Const or do we want it modifyable?

Reply:
This provides the user the possibility to modify the template for customization purposes. We do the same for the other templates so I thought this would be reasonable, WDYT?

Reply:
Sure as long as it's a conscious decision. Perhaps dependency injection would be better, but life is hard and then we all die.

Reply:
> Sure as long as it's a conscious decision. Perhaps dependency injection would be better, but life is hard and then we all die.

I keep that in mind for a later improvement...the dependency injection, not the fact that we all will die.